test/framework/src/main/java/org/elasticsearch/cluster/DiskUsageIntegTestCase.java

```diff
@@ -114,11 +114,11 @@ public String name() {
 
         @Override
         public long getTotalSpace() throws IOException {
-            final long totalSpace = this.totalSpace;
-            if (totalSpace == -1) {
+            final long totalSpaceCopy = this.totalSpace;
```

Comment:
I don't quite see the point is creating this intermediate variable in the first place.

Reply:
It's a `volatile` variable, I assume the copy is something to do with that.